lib/asciidoctor/pdf/converter.rb

```diff
@@ -933,9 +933,17 @@ def convert_open node
       # TODO process abstract child even when partintro has multiple blocks
       convert_abstract node.blocks[0]
     else
-      add_dest_for_block node if node.id
-      layout_caption node.title if node.title?
-      convert_content_for_block node
+      if node.has_role? 'keep-together'
```

Comment:
To be consistent with AsciiDoc.py and other converters, this should instead check for the unbreakable role.

```rb
if node.option? 'unbreakable'
```

Reply:
> this should instead check for the unbreakable role.

You mean option?

> You can use the unbreakable option to instruct the browser not to break a block element. The following image and it’s caption will be kept together the printed page:
> ```
> [options="unbreakable"]
> .Tiger block image
> image::images/tiger.png[Tiger image]
> ```
https://www.methods.co.nz/asciidoc/faq.html#_how_can_i_control_page_breaks_when_printing_html_outputs

Reply:
Yep! Or the shorthand for it:

```
[%unbreakable]
```